app/styles/globals.css

```diff
@@ -14,6 +14,12 @@ html {
   font-size: var(--font-size-md);
 }
 
+button {
+  cursor: pointer;
+  border: none;
+  background-color: transparent;
+}
```

Comment:
With this, do we still need https://github.com/webkom/lego-webapp/pull/4975/files? Also, very nitpick but can we move this to after the `body` selector so that there's some sort of hierarchy here

Reply:
Is this global.css file included in lego-bricks? The button styling should work for other projects using lego-bricks as well.

Reply:
Should it though? Is it not just lego-webapp specific to use regular `button`s like we do? Not a fan of component libraries that add global styling

Reply:
Not really, but that would break the styling of e.g. the `lego-editor` demo @ivarnakken

Reply:
Ok, I agree `lego-bricks` shouldn't add any global styling. But do the buttons in `lego-bricks` look good even without this? If they do, and this is just for the non-`lego-bricks` buttons, I think this is good:)